.github/workflows/trivy.yml

```diff
@@ -0,0 +1,38 @@
+name: Trivy
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "30 7 * * *"
```

Comment:
Any specific reason this is scheduled for 1:00 PM IST? :) We usually run automated workflows around midnight (off-peak hours) to minimize load during development hours. Shall we align this one the same way?